src/server/views/help.pug

```diff
@@ -0,0 +1,54 @@
+extends includes/layout.pug
+block title
+    title Help - NetsBlox
+block content
+  mixin box(title, url)
+    .col-md-4.col-sm-6.text-center
+      .box
+        h2.tag-title= title
+        block          
+        br
+        a.btn.btn-block.btn-primary(href=url) Learn more
```

Comment:
Some of these could be "Learn more" or "More" but others (like the flyer) are not exactly fitting "learn more" and make more sense with something like "View" or "Check it out" or something